build.sbt

```diff
@@ -334,17 +369,19 @@ def sharedNativeImageSettings(targetDir: File) = Seq(
     "akka.dispatch.affinity",
     "akka.util",
     "com.google.Protobuf",
-    "com.typesafe.config",
-    "java.lang.ref.SoftReference", // https://github.com/oracle/graal/issues/2345
-    "java.lang.invoke.MethodHandleImpl" // https://github.com/oracle/graal/issues/2345
+    "scala.runtime.Statics$VM"
```

Comment:
I would've thought this was redundant with the `"scala"` above.

Reply:
@dwijnand I'll see if that's the case